dispatch/static/manager/src/js/components/EventEditor/EventForm.js

```diff
@@ -0,0 +1,170 @@
+            value={this.props.listItem.description || ''}
+            fill={true}
+            onChange={ e => this.props.update('description', e.target.value) } />
+        </FormInput>
+
+        <FormInput
+          label='Host'
+          padded={false}
+          error={this.props.errors.host}>
+          <TextInput
+            placeholder='Host'
+            value={this.props.listItem.host || ''}
+            fill={true}
+            onChange={ e => this.props.update('host', e.target.value) } />
+        </FormInput>
+
+        <Dropzone
+          ref={(node) => { this.dropzone = node }}
+          className='c-person-form__image__dropzone'
+          onDrop={(files) => this.onDrop(files)}
+          disableClick={true}
+          activeClassName='c-person-form__image__dropzone--active'
+          multiple={false}>
+          <div
+            className='c-person-form__images__container'>
+            {this.state.displayImg || this.props.listItem.image ? null :
+              <div className='c-person-form__image__dropzone__text'>
+                Drop Image Here
+              </div>}
+            <img
+              className='c-person-form__images'
+              src={this.state.displayImg || this.props.listItem.image}/>
+          </div>
+        </Dropzone>
+
+        <div className='c-person-form__image__button'>
+          <AnchorButton
+            onClick={() => this.dropzone.open()}>Select Image
+          </AnchorButton>
+        </div>
+
+        <FormInput
+          label='Start Time'
+          padded={false}
+          error={this.props.errors.start_time}>
+          <DateTimeInput
+            value={this.ensureDate(this.props.listItem.start_time)}
+            onChange={ dt => this.props.update('start_time', dt)} />
+        </FormInput>
+
+        <FormInput
+          label='End Time'
+          padded={false}
+          error={this.props.errors.end_time}>
+          <DateTimeInput
+            value={this.ensureDate(this.props.listItem.end_time)}
+            onChange={ dt => this.props.update('end_time', dt)} />
+        </FormInput>
+
+        <FormInput
+          label='Location'
+          padded={false}
+          error={this.props.errors.location}>
+          <TextInput
+            placeholder='Location'
+            value={this.props.listItem.location || ''}
+            fill={true}
+            onChange={ e => this.props.update('location', e.target.value) } />
+        </FormInput>
+
+        <FormInput
+          label='Address'
+          padded={false}
+          error={this.props.errors.address}>
+          <TextInput
+            placeholder='Address'
+            value={this.props.listItem.address || ''}
+            fill={true}
+            onChange={ e => this.props.update('address', e.target.value) } />
+        </FormInput>
+
+        <FormInput
+          label='Category'
+          padded={false}
+          error={this.props.errors.category}>
+          <SelectInput
```

Comment:
This field doesn't seem to be working properly. It doesn't show the right category when editing an existing event